README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository is designed to manage and process events related to changes in wallet status within the PagoPA payment wallet ecosystem. This service listens to events that signal changes (such as wallet creation, modification, or deletion) and ensures that these updates are appropriately handled in a (near) real-time manner.
 
-- [PagoPA Wallet CDC Service](#pagopa-payment-wallet-cdc-service)
+- [PagoPA Wallet CDC Service](#pagopa-wallet-cdc-service)
```

Comment:
the repo name is pagopa-payment-wallet-cdc-service, why rename?

Reply:
corrected